src/index.js

```
@@ -1 +1,29 @@
-'use strict';
+require('dotenv').config();
+
+const express = require('express');
+const { authRouter } = require('./routes/auth.route.js');
+const cors = require('cors');
+const { errorMiddleware } = require('./middlewares/errorMiddleware.js');
+const cookieParser = require('cookie-parser');
+
+const server = express();
+
+server.use(express.json());
+server.use(cookieParser());
+
+server.use(
+  cors({
+    origin: process.env.CLIENT_HOST,
+    credentials: true,
+  }),
+);
+
+server.use(authRouter);
+
+server.use((_, res) => {
+  res.status(404).json({ message: 'Not Found' });
+});
+
+server.use(errorMiddleware);
+
+server.listen(process.env.PORT);
```

Practical improvement (recommended): `server.listen(process.env.PORT)` will try to use the environment value directly. If PORT is undefined or a string you'd like to coerce to number, provide a default and log the listening port. For example, use `const PORT = parseInt(process.env.PORT, 10) || 3000; server.listen(PORT, () => console.log(`Server listening on ${PORT}`));`. This prevents silent failures or unclear startup behavior.